recipe/meta.yaml

```diff
@@ -1,23 +1,23 @@
-{% set python_min = '3.10' %}
+{% set python_min = "3.10" %}
 {% set name = "jax" %}
-{% set version = "0.4.36" %}
+{% set version = "0.4.37" %}
 # Not declared in the metadata but essential to this package.
-# Update _minimum_jaxlib_version from https://github.com/jax-ml/jax/blob/main/jax/version.py
+# Update from _minimum_jaxlib_version in https://github.com/jax-ml/jax/blob/main/jax/version.py
 {% set minimum_jaxlib_version = "0.4.36" %}
```

Comment:
For all recent versions the minimum is exactly `version` (so this line is wrong for 0.4.37, see [here](https://github.com/jax-ml/jax/blob/jax-v0.4.37/jax/version.py#L24)). I don't see a reason to make this distinction TBH, just pin 
```yaml
- jaxlib =={{ version }}
```
below

Reply:
Alright, I'm closing. I don't see a reason why not (since jaxlib is updated first and matched), so okay.